plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeConfig.java

```diff
@@ -125,6 +127,7 @@ public void testExplicitPropertyMappings()
                 .setUnsafeWritesEnabled(true)
                 .setDefaultCheckpointWritingInterval(15)
                 .setCheckpointRowStatisticsWritingEnabled(false)
+                .setCheckpointPartitionFilterEnabled(true)
```

Comment:
keep accessors & config in same order as the fields in the config class (eg add after defaultCheckpointWritingInterval everywhere)